Answers/ho03_htook.py

```diff
@@ -0,0 +1,41 @@
+#this is where i am coding
+import turtle
+kh1 = turtle.Turtle
+wn = turtle.Screen()
+wn.bgcolor('lightblue')
+
+kh1.penup()
+kh1.setposition(200,100)
+kh1.pendown()
+kh1.color('yellow')
+kh1.pensize(100)
+kh1.circle(5)
```

Comment:
There should be no code like this outside of `main` or a function. Move this code to the start of your `main`.